src/api/app/views/webui2/webui/request/_show_side_links.html.haml

```diff
@@ -0,0 +1,45 @@
+%ul.list-unstyled
+  %li
+    %i.fas.fa-user.text-success
+    Created
+    by #{user_with_realname_and_icon(request_creator, no_icon: true, short: true)} #{fuzzy_time(request_created_at)}
```

Comment:
Please put the Ruby code on another line, you won't have to interpolate it in the string.

```
Created by
= user_with_realname_and_icon(request_creator, no_icon: true, short: true) 
= fuzzy_time(request_created_at)
```

Reply:
done, in some another places too